### PR TITLE
DDCE-4416 : Add 'In pounds' to the actual income field

### PR DESCRIPTION
**Pull request by Divya-Manivannan-hmrc**

### CODE REVIEW

File: conf/messages.en [hunk 1]
```diff
@@ -920,7 +920,7 @@ estimatedIncome.checkYourAnswersLabel = Estimated income for the current 12 mont
 # What is the charity’s actual income so far?
 # ----------------------------------------------------------
 actualIncome.title = What is the charity’s actual income so far?
-actualIncome.heading = What is the charity’s actual income so far?
+actualIncome.heading = What is the charity's actual income in pounds so far?
```
Comment (EdCope) [inline conf/messages.en:923]: Maintain the correct apostrophe `’` and also adjust the title as the content should match.

You will see when you test that the current one used isn't displayed. Please provide screenshot for design sign-off too.

File: project/AppDependencies.scala [hunk 2]
```diff
@@ -3,12 +3,12 @@ import sbt.*
...
 
   private lazy val mongoHmrcVersion     = "2.2.0"
-  private lazy val bootstrapPlayVersion = "9.2.0"
+  private lazy val bootstrapPlayVersion = "9.3.0"
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % mongoHmrcVersion,
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "10.6.0"
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "10.8.0"
```
Comment (EdCope) [inline project/AppDependencies.scala:11]: Please update to `10.9.0` when making the correction

**Review by EdCope — CHANGES_REQUESTED** [review]

File: conf/messages.en (hunk 1 above)
Comment (Divya-Manivannan-hmrc) [reply conf/messages.en:923]: Done

File: project/AppDependencies.scala (hunk 2 above)
Comment (Divya-Manivannan-hmrc) [reply project/AppDependencies.scala:11]: Done

**Comment by EdCope** [comment]
Please amend the acceptance tests to reflect this change

**Pull request merged**